app/src/main/java/ca/klapstein/baudit/data/Record.java

```diff
@@ -25,26 +25,30 @@
     private GeoLocation geoLocation;
     private ArrayList<BodyPhotoCoords> bodyPhotoCoords = new ArrayList<>();
     private ArrayList<String> keywords = new ArrayList<>();
-    private UUID recordId;
+    private UUID recordID;
```

Comment:
Is `problemId` renamed aswell for consistency?

Reply:
In camelCase, it is common practice to keep only the first letter of an acronym capitalized.
e.g. problemId, recordId, parsedHtml, etc.